config/locales/client.en.yml

```diff
@@ -380,17 +379,15 @@ en:
     explain_scope:
       thread: This changes when you'll be notified about this thread
       group: This changes when you'll be notified about all threads in the group
-
     simple:
       label: Notifications
       loud: Subscribed
       normal: Normal
       quiet: Ignore
       question: When do you want to be notified?
-      loud_explain: whenever there is activity
-      normal_explain: when someone asks for your attention
-      quiet_explain: no notifications
-
+      loud_explain: Whenever there is activity
```

Comment:
This is composed into a larger string, so that's why it begins with lower case. I would not do that now, but this was done a long time ago.